contrib/localnet/scripts/start-zetaclientd.sh

```diff
@@ -41,10 +41,10 @@ while [ ! -f ~/.ssh/authorized_keys ]; do
     sleep 1
 done
 
-# need to wait for zetacore0 to be up
-while ! curl -s -o /dev/null zetacore0:26657/status ; do
-    echo "Waiting for zetacore0 rpc"
-    sleep 20
+# need to wait for zetacore0 block 1 to be available
+while curl -s zetacore0:26657/block?height=1 | jq -e '.error != null' > /dev/null; do
```

Comment:
Consider refactoring to a function which accepts a height 
`wait_for_block()`

Reply:
We only do this once so I don't see a strong reason to make a function.

Reply:
Don't we do something similar when waiting for the upgrade height? Its a different script but we can maintain a set of common functions .

Reply:
Yeah we could make some shared library functions. https://github.com/zeta-chain/node/issues/3420

Reply:
Sounds good ,we can do that in a separate pr 
Please add a TODO , if thats the plan